policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/childrens_fitness_tax_credit/base.yaml

```
@@ -0,0 +1,9 @@
+description: Yukon child fitness fee $1,000 for eligible fees paid for the cost of registration or membership.
```

Phrase other descriptions similarly, and avoid including the parameter value in the description or label
```suggestion
description: Yukon provides up to this amount toward the cost of registration or membership in children's fitness programs.
```